pg_auto_failover.spec

```diff
@@ -0,0 +1,66 @@
+%global pgmajorversion 11
+%global pgpackageversion 11
+%global pginstdir /usr/pgsql-%{pgpackageversion}
+%global sname pg_auto_failover
+%global extname pgautofailover
+
+Summary:	todo
```

Comment:
needs to be filled out once we have our 1 liner

Reply:
Postgres extension for automated failover and high-availability